[2025-07-25T21:34:57.988+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-25T21:34:58.010+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sigtap_00_installation.gerar_schema_sql manual__2025-07-25T21:34:53.396548+00:00 [queued]>
[2025-07-25T21:34:58.016+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sigtap_00_installation.gerar_schema_sql manual__2025-07-25T21:34:53.396548+00:00 [queued]>
[2025-07-25T21:34:58.016+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-07-25T21:34:58.028+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): gerar_schema_sql> on 2025-07-25 21:34:53.396548+00:00
[2025-07-25T21:34:58.035+0000] {warnings.py:112} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=59) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-07-25T21:34:58.036+0000] {standard_task_runner.py:63} INFO - Started process 60 to run task
[2025-07-25T21:34:58.036+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'sigtap_00_installation', 'gerar_schema_sql', 'manual__2025-07-25T21:34:53.396548+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/sigtap_00_installation_dag.py', '--cfg-path', '/tmp/tmpyg4vrkzt']
[2025-07-25T21:34:58.037+0000] {standard_task_runner.py:91} INFO - Job 5: Subtask gerar_schema_sql
[2025-07-25T21:34:58.089+0000] {task_command.py:426} INFO - Running <TaskInstance: sigtap_00_installation.gerar_schema_sql manual__2025-07-25T21:34:53.396548+00:00 [running]> on host 012b9571e725
[2025-07-25T21:34:58.165+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='sigtap_00_installation' AIRFLOW_CTX_TASK_ID='gerar_schema_sql' AIRFLOW_CTX_EXECUTION_DATE='2025-07-25T21:34:53.396548+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-25T21:34:53.396548+00:00'
[2025-07-25T21:34:58.166+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-25T21:34:58.178+0000] {subprocess.py:63} INFO - Tmp dir *** location: /tmp
[2025-07-25T21:34:58.179+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/airflow/src/Step_01_generate_schema_db_sigtap_sql.py']
[2025-07-25T21:34:58.190+0000] {subprocess.py:86} INFO - Output:
[2025-07-25T21:34:58.297+0000] {subprocess.py:93} INFO - 
[2025-07-25T21:34:58.298+0000] {subprocess.py:93} INFO - Erro Crítico: A pasta base de layouts não foi encontrada em '/downloads/unzipped'.
[2025-07-25T21:34:58.298+0000] {subprocess.py:93} INFO - Processo abortado.
[2025-07-25T21:34:58.302+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-07-25T21:34:58.303+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-25T21:34:58.325+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=sigtap_00_installation, task_id=gerar_schema_sql, run_id=manual__2025-07-25T21:34:53.396548+00:00, execution_date=20250725T213453, start_date=20250725T213458, end_date=20250725T213458
[2025-07-25T21:34:58.370+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-25T21:34:58.390+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-07-25T21:34:58.391+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
